src/main/resources/locale/locale_de.properties

```
@@ -201,8 +201,8 @@ Commands.Party.Alliance.Ally = &f{0} &8Ist verb\u00FCndet mit: &f{1}
 Commands.Party.Alliance.AlreadyAllies = Deine Party ist bereits in einem B\u00FCndnis. Trenne es mit&3/party alliance disband
 Commands.Party.Alliance.Header = -----[]&aParty-B\u00FCndnisse&c[]-----
 Commands.Party.Alliance.Help.0 = Diese Party ist in keinem B\u00FCndnis. Lade einen Party-Anf\u00FChrer ein.
-Commands.Party.Alliance.Help.1 = &c um zu verb\u00FCnden &3/party alliance invite <Spieler>&c.
-Commands.Party.Alliance.Invite.0 = ALERT: &aDu hast eine B\u00FCndnis-Anfrage f\u00FCr {0} von {1} erhalten.
+Commands.Party.Alliance.Help.1 = &c Um zu verb\u00FCnden: &3/party alliance invite <Spieler>&c.
+Commands.Party.Alliance.Invite.0 = Warnung: &aDu hast eine B\u00FCndnis-Anfrage f\u00FCr {0} von {1} erhalten.
```

I think removing the caps here takes away from the artistic choice.
```suggestion
Commands.Party.Alliance.Invite.0 = ACHTUNG: &aDu hast eine B\u00FCndnis-Anfrage f\u00FCr {0} von {1} erhalten.
```